src/neuroglancer/sliceview/decode_swc_skeleton.ts

```diff
@@ -0,0 +1,77 @@
+import {each} from 'lodash';
+import {decodeSkeletonVertexPositionsAndIndices, ParameterizedSkeletonSource, SkeletonChunk} from 'neuroglancer/skeleton/backend';
+import {convertEndian32, Endianness} from 'neuroglancer/util/endian';
+
+export function decodeSwcSkeletonChunk(
+    chunk: SkeletonChunk, swc_str: string, endianness: Endianness) {
```

Comment:
Yeah, that's right.  Endian conversion is only needed for converting binary formats into native endianness.